fdbserver/ConsistencyScan.actor.cpp

```diff
@@ -992,10 +992,12 @@ ACTOR Future<Void> checkDataConsistency(Database cx,
 				    .detail("NumSampledKeys", sampledKeys)
 				    .detail("NumSampledKeysWithProb", sampledKeysWithProb);
 
+				// NOTE: Shard sampling is known to be biased.
+				// Downgrade this to a warning until we have a proper solution.
 				testFailure(format("Shard size is more than %f std dev from estimate", failErrorNumStdDev),
 				            performQuiescentChecks,
 				            success,
-				            failureIsError);
+				            /*failureIsError*/ false);
```

Comment:
Is the effect of this that the check will be retried? If so, I think we may need to remove the call to `testFailure` entirely since this isn't a condition that will be repaired for later retries

Reply:
Yes, you're right. It appears that `testFailure` will trigger a retry, which we don't want. I've commented it out entirely and changed the above TraceEvent to a warning instead.